publisher/server/publisher.go

```diff
@@ -22,9 +22,8 @@ const (
 type Server struct {
 	DockerClient *docker.Client
```

Comment:
good point, but that's unrelated to the PR. Definitely agree that we should make or use some form of interface here.